MAINTAINING.md

```diff
@@ -0,0 +1,212 @@
+   arrives for both, it will send 2 PRs, where they should both be upgraded at the same time.
+   
+2. To overcome those disadvantages, we have our own dependency upgrade solution based on the
+   [Gradle Use Latest Versions Plugin](https://github.com/patrikerdes/gradle-use-latest-versions-plugin). It runs daily
+   during weekdays.
+   
+When an upgrade to a new version arrives, we need to be careful when merging, so that we don't introduce an
+unnecessary upgrade burden on our users. Read the
+[Module Upgrade Strategy](https://github.com/micronaut-projects/micronaut-core/wiki/Module-Upgrade-Strategy) for more
+information.
+
+Note that if a new version arrives and we are not ready yet to do the upgrade, you need to
+[pin the old version](https://github.com/micronaut-projects/micronaut-build/#configuration-options), because otherwise,
+Dependabot and our workflow will keep sending PRs. You should also create an issue to upgrade so that it's not forgotten.
+
+#### Files sync
+
+We have a [template repo](https://github.com/micronaut-projects/micronaut-project-template) that we use as the single
+source of truth for certain files. It is used as a template to create new repos, and changes to certain files in the
+temaplate repo will get propagated automatically. The files propagated are:
+
+* Workflow files (`.github/workflows/*`). They are copied using rsync"
+  * `bintray-publish.yml`.
+  * `central-sync.yml`.
+  * `dependency-update.yml`.
+  * `graalvm.yml`.
+  * `gradle.yml`.
+  * `release.yml`.
+  * `release-notes.yml`.
+* Dependabot configuration (`.github/dependabot.yml`).
+* Gradle wrapper.
+* `.gitignore`.
+* `ISSUE_TEMPLATE.md`, `LICENSE`, `MAINTAINING.md`, `config/HEADER` and `config/spotless.license.java`.
+* Checkstyle's `config/checkstyle/checkstyle.xml` and `config/checkstyle/suppressions.xml`.
+
+Regarding the Gradle wrapper, the template repo checks every week if there is a new Gradle version. If there is, it will
+upgrade the wrapper in the template repo itself, and via the files sync workflow this gets propagated to all repos. This
+way we make sure we stay up-to-date regarding Gradle versions in all repos.
+
+##### Customised workflow files
+
+Due to limitations in the GitHub Actions design (such that they don't allow including snippets or any other kind of
+reusability), for the sync'ed workflow files listed above, it is not possible to have custom steps and still be part of
+the sync process, since any modification to those files will be overwritten the next time the files sync workflow is
+executed.
+
+The "Java CI" (`gradle.yml`) workflow does have the ability to have an optional setup step, though. If there is a `setup.sh`
+file in the project root, it will be executed before invoking Gradle.
+
+There are projects, such as micronaut-gcp and micronaut-kubernetes, that have made customisations to sync'ed workflows
```

Comment:
`customisations` is the British spelling, as opposed to `customizations`, Is that what you want?

Reply:
Correct

Reply:
Correct that we're using British spellings? I think we should be consistent and use American English everywhere.

Reply:
I learned British English and have always written British English to the best of my knowledge. I also doubt that 100% of the existing Micronaut documentation is written in American English.

That said, I'm not against having this or any other document written in American English, so feel free to send a PR updating those.